### PR TITLE
docs(adapters): update copy regarding adapters

### PR DESCRIPTION
**Pull request by ndom91**
## Reasoning 💡

Mentioned the `nextauthjs/adapters` repo in appropriate places and updated the docs to reflect the current reality about adapters.

## Checklist 🧢

- [x] Documentation
~- [ ] Tests~
- [x] Ready to be merged

### CODE REVIEW

File: www/docs/configuration/databases.md [hunk 1]
```diff
@@ -5,13 +5,17 @@ title: Databases
...
-  _The TypeORM adapter supports MySQL, Postgres, MsSql, SQLite and MongoDB databases._
-* **Prisma**<br/>
-  _The Prisma 2 adapter supports MySQL, Postgres and SQLite databases._
-* **Custom Adapter**<br/>
+- **TypeORM** (default)<br/>
+  _The TypeORM adapter supports MySQL, PostgreSQL, MSSQL, SQLite and MongoDB databases._
+- **Prisma**<br/>
+  _The Prisma 2 adapter supports MySQL, PostgreSQL and SQLite databases._
+- **Fauna**<br/>
+  _The FaunaDB adapter only support FaunaDB._
```
Comment (balazsorban44) [inline www/docs/configuration/databases.md:13]: supports*

File: www/docs/configuration/databases.md [hunk 2]
```diff
@@ -5,13 +5,17 @@ title: Databases
...
+- **Custom Adapter**<br/>
   _A custom Adapter can be used to connect to any database._
```
Comment (balazsorban44) [inline www/docs/configuration/databases.md:16]: I, feel like we will need to generate a list of adapters the same way we do with providers at some point, may be OK for now.

File: www/docs/schemas/adapters.md [hunk 3]
```diff
@@ -7,6 +7,24 @@ An **Adapter** in NextAuth.js connects your application to whatever database or
 
 You do not need to specify an Adapter explicitly unless you want to use advanced options such as custom models or schemas, if you want to use the Prisma Adapter instead of the default TypeORM Adapter, or if you are creating a custom Adapter to connect to a database that is not one of the supported databases.
 
+We have recently begun moving the adapter code out of the core of `nextauthjs/next-auth` and into their own repository under [`nextauthjs/adapters`](https://github.com/nextauthjs/adapters).
```
Comment (balazsorban44) [inline www/docs/schemas/adapters.md:10]: I think the migration is done, so I would rephrase this so we won't need to revisit this sentence later on. Something like adapters are separate from core and they can be found under...

File: www/docs/schemas/adapters.md [hunk 4]
```diff
@@ -7,6 +7,24 @@ An **Adapter** in NextAuth.js connects your application to whatever database or
...
+
+There you can find the following adapters in various stages of maturity:
+
+- TypeORM
+- TypeORM-Legacy
```
Comment (balazsorban44) [inline www/docs/schemas/adapters.md:15]: There is only TypeORM, I guess the name is confusing, but the indication is that we might want to clean up the typeorm adapter later in, but leave the old one as is. Prisma has two versions, so that's OK.

File: www/docs/schemas/adapters.md [hunk 5]
```diff
@@ -7,6 +7,24 @@ An **Adapter** in NextAuth.js connects your application to whatever database or
...
+- Prisma
+- Prisma-Legacy
+- FaunaDB
+- DynamoDB
+- PouchDB (Active PR)
+- Sanity (Active PR)
+- Sequelize (Active PR)
+
```
Comment (balazsorban44) [inline www/docs/schemas/adapters.md:23]: Auto-generated this list somehow? Have a single source of truth for adapters somewhere.

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

File: www/docs/configuration/databases.md (hunk 2 above)
Comment (ndom91) [reply www/docs/configuration/databases.md:16]: Yeah probably :/

File: www/docs/schemas/adapters.md (hunk 5 above)
Comment (ndom91) [reply www/docs/schemas/adapters.md:19]: Yeah probably also want to autogenerate this eventnually. I addressed all your comments and actually removed the ones I had listed here with (Active PR) because that stuff can change so quickly.. I just kept the actually active ones on that list for now - which made it shorter and easier to deal with manually for now anyway :+1:

**Comment by balazsorban44** [comment]
Would like to wait with this until we had a chat about https://github.com/nextauthjs/adapters/pull/94

File: www/docs/schemas/adapters.md [hunk 6]
```diff
@@ -7,6 +7,20 @@ An **Adapter** in NextAuth.js connects your application to whatever database or
...
+- TypeORM
+- Prisma
+- Prisma-Legacy
+- FaunaDB
+- DynamoDB
+
+## TypeORM Adapter
+
+NextAuth.js comes with a default Adapter that uses [TypeORM](https://typeorm.io/) so that it can be used with many different databases without any further configuration, you simply add the node module for the database driver you want to use to your project and pass a database connection string to NextAuth.js.
```
Comment (lluia) [inline www/docs/schemas/adapters.md:22]:
```suggestion
NextAuth.js comes with a default Adapter that uses [TypeORM](https://typeorm.io/) so that it can be used with many different databases without any further configuration, you simply add the node module for the database driver you want to use in your project and pass a database connection string to NextAuth.js.
```
Super minor but I think this is the correct spelling in English 🤔 ✍🏼

**Review by 0ubbe — APPROVED** [review]
Coolios for me 💯 ( understanding we're still waiting on #94 )

File: www/docs/schemas/adapters.md (hunk 6 above)
Comment (ndom91) [reply www/docs/schemas/adapters.md:22]: Yup you're correct :) I'm a native english speaker and I even missed that :facepalm: haha
